ruby-training/trainees/alan/test_katas/01/math_maddness/spec/math_maddness_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rspec/autorun'
+
+class Aritmetic_operations
+    
+    def self.add(first_number, second_number)
+       first_number +  second_number
+    end
+    
+    def self.substraction(first_number, second_number)
+        first_number - second_number
+    end
+
+    def self.multiply(first_number, second_number)
+        first_number * second_number
+    end
+
+    def self.divide(first_number, second_number)
+        first_number / second_number
+    end
+end
+describe Aritmetic_operations do
+    describe "#add" do
+        it "Sum the two numbers and the result is equal to 16" do
```

Comment:
This line should be readable as an english language sentence. Also, the text in `it` blocks is printed to the console as the middle of a sentence (i.e. "ArithmeticOperations .add should return the sum of two numbers"), so you should use lowercase at the beginning as well. 

Finally, this description is too specific, I should be able to tell "what" the spec does without having to know the "how" details (like "and the result is equal to 16"). If I need details, I should be able to get them only by reading the code inside the block. Something like the example in the paragraph above would be better (describe behavior, not specific results).